org.gnome.Music.json

```diff
@@ -88,34 +81,35 @@
             ],
             "sources": [{
                 "type": "archive",
-                "url": "https://download.gnome.org/sources/grilo/0.3/grilo-0.3.4.tar.xz",
-                "sha256": "7c6964053b42574c2f14715d2392a02ea5cbace955eb73e067c77aa3e43b066e"
+                "url": "https://download.gnome.org/sources/grilo/0.3/grilo-0.3.6.tar.xz",
+                "sha256": "1e65ca82dd58020451417fde79310d4b940adc3f63ab59997419c52ed3bc9c91"
             }],
             "cleanup": ["/include", "/bin"]
         },
         {
             "name": "grilo-plugins",
+            "buildsystem": "meson",
             "config-opts": [
-                "--enable-tracker",
-                "--enable-dleyna",
-                "--enable-goa",
-                "--enable-filesystem=no",
-                "--enable-optical-media=no",
-                "--enable-youtube=no",
-                "--enable-bookmarks=no",
-                "--enable-lua-factory=no",
-                "--enable-metadata-store=no",
-                "--enable-vimeo=no",
-                "--enable-localmetadata=no",
-                "--enable-thetvdb=no",
-                "--enable-tmdb=no",
-                "--enable-freebox=no",
-                "--enable-opensubtitles=no"
+                "-Denable-tracker=yes",
+                "-Denable-dleyna=yes",
+                "-Denable-goa=yes",
```

Comment:
This is no longer a build-time switch (it is a hidden option, should probably report this upstream).